packages/fscomponents/src/components/Step.tsx

```diff
@@ -53,6 +53,9 @@ export class Step extends Component<StepProps> {
         <TouchableOpacity
           onPress={this.props.onPress}
           style={[styles.container, this.props.style]}
+          accessibilityTraits='button'
+          accessibilityComponentType='button'
+          accessibilityLabel={`${this.props.title}, Step Completed`}
```

Comment:
@KoHsinChuBB "Step Completed" should be added to the `fsi18n` package and pulled from there.

Reply:
@justin-endler  Will fix it, thanks.